elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/ElideConfigParser.java

```diff
@@ -72,12 +72,14 @@ private void parse(String localConfigPath) throws Exception {
 
         //security
         String securityJson = util.getJsonConfig(localConfigPath, DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY).get(0);
-        populatePojo(securityJson, DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY);
+        populatePojo(util.resolveVariables(securityJson, this.variables),
+                DynamicConfigHelpersUtil.SCHEMA_TYPE_SECURITY);
 
         //table
         Set<Table> tables = new HashSet<>();
         for (String tableJson : util.getJsonConfig(localConfigPath, DynamicConfigHelpersUtil.SCHEMA_TYPE_TABLE)) {
-            ElideTableConfig table = (ElideTableConfig) parseJsonConfig(tableJson,
+            ElideTableConfig table = (ElideTableConfig) parseJsonConfig(
```

Comment:
We should store the output of `util.resolveVariables(tableJson, this.variables)` in a variable and then call populatePojo method.

Reply:
resolved in https://github.com/yahoo/elide/pull/1210